docs/configuration/install.md

```diff
@@ -225,14 +232,9 @@ systemctl restart httpd.service
 <a name="php-installationconfiguration"></a>
 ## PHP Installation/Configuration
 
-On Debian/Ubuntu 14.04 execute:
-```
-apt-get install libapache2-mod-php5
-```
-
-On Ubuntu 16.04 execute:
+On Debian/Ubuntu 14.04 or 16.04 execute:
 ```
-apt-get install libapache2-mod-php7.0
+apt-get install libapache2-mod-php7.1 php7.1-gd php7.1-mysql php7.1-mbstring php-xsl
```

Comment:
php-xsl ?? xsl is the name of the extension but I think the package required is php-xml

Reply:
At least in Ubuntu php-xml seems to include everything we need (dom, xsl, xml, ...)